js/legends.ts

```
@@ -36,133 +70,67 @@ export class CosmographLegends {
-    let container: HTMLDivElement | undefined
-    let legendInstance: CosmographSizeLegend | CosmographRangeColorLegend | CosmographTypeColorLegend | undefined
     switch (`${type}_${property}`) {
       case 'point_size':
-        container = this.pointSizeLegendContainer
-        if (!hide && !this._pointSizeLegend) {
-          this._pointSizeLegend = new CosmographSizeLegend(this.cosmograph, container, {
-            label: d => `${type}s by ${d}`,
-          })
-        }
-        legendInstance = this._pointSizeLegend
+        const activePointSizeStrategy = this.cosmograph.activePointSizeStrategy
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)